addon/inbound-actions.js

```diff
@@ -3,9 +3,16 @@ import ActionProxy from './action-proxy';
 
 export default Em.Mixin.create({
   _inbound_actions_setup: Em.on('init', function() {
-    Em.run.schedule('afterRender', this, function() {
+    let scheduleInAfterRender = /^1\.13|^[2-9]/.test(Em.VERSION);
+
+    if(scheduleInAfterRender) {
+      Em.run.schedule('afterRender', this, function() {
+        var proxy = ActionProxy.create({ target: this });
```

Comment:
Thanks, I was trying to avoid polluting the consuming object with additional functions, but I think it's worth it. I'll update